instrumentation/apache-camel-2.20/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/apachecamel/CamelRoutePolicy.java

```diff
@@ -51,6 +51,12 @@ private Span spanOnExchangeBegin(Route route, Exchange exchange, SpanDecorator s
     return builder.startSpan();
   }
 
+  private Span.Kind spanKind(SpanDecorator sd) {
+    Span activeSpan = CamelTracer.TRACER.getCurrentSpan();
+    // root operation, set kind, otherwise - INTERNAL
```

Comment:
It took me a while to grok this, how about `// if there's an active span, this is not a root span which we always mark as INTERNAL`

Reply:
+1